kotlinx-coroutines-core/api/kotlinx-coroutines-core.api

```diff
@@ -30,6 +30,11 @@ public final class kotlinx/coroutines/BuildersKt {
 	public static final fun withContext (Lkotlin/coroutines/CoroutineContext;Lkotlin/jvm/functions/Function2;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 }
 
+public final class kotlinx/coroutines/Builders_concurrentKt {
```

Comment:
I would assume that it's possible to workaround the need for that middle layer by adding
```
@file:JvmMultifileClass
@file:JvmName("BuildersKt")
```

to `Builders.concurrent.kt`. Could you please consider doing so?